### PR TITLE
Fix ETL bug with CRIS v21.0 update

### PR DESCRIPTION
**Pull request by mateoclarke**
Closes https://github.com/cityofaustin/atd-data-tech/issues/3982

As described in the issue, a UI update on TXDOT's end broke our Extract request script.

At first the issue was that the script was expecting the buttons to say "Continue", but now they say "Next". There were also updates to the way radio and select option values are selected.

I've tested this on staging and it looks successful. Along with some screenshots before stepping through to the next page.

```
➜  atd-etl git:(3982_fix_etl_bug) ✗ runetl ~/.ssh/atd-etl/etl.staging.env app/process_cris_request.py
Using environment variables in '/Users/mateoclarke/.ssh/atd-etl/etl.staging.env'...


----- ETL RUN ------
Run Environment:        staging
Run File:               app/process_cris_request.py
ATD_CRIS_CONFIG:        /Users/mateoclarke/.ssh/atd-etl/etl.staging.env
ATD_DOCKER_IMAGE:       atddocker/atd-vz-etl:local
--------------------

--------------------------------------------------------------------
--- Running web-pdb debugger at http://localhost:5555 ---
--------------------------------------------------------------------
docker run -it --rm -p 5555:5555/tcp         -v /Users/mateoclarke/Desktop/code/CoA/ATD/atd-vz-data/atd-etl/app:/app -v /Users/mateoclarke/Desktop/code/CoA/ATD/atd-vz-data/atd-etl/data:/data -v /Users/mateoclarke/Desktop/code/CoA/ATD/atd-vz-data/atd-etl/tmp:/app/tmp         --env-file /Users/mateoclarke/.ssh/atd-etl/etl.staging.env atddocker/atd-vz-etl:local /app/process_cris_request.py;
--------------------------------------------------------------------
Initializing browser options.
Initializing Chrome headless browser.
Logging in to 'https://cris.dot.state.tx.us/'
Filling out agency.
Filling out credentials.
Selecting data extract request, from '09/18/2020' to '09/21/2020'
Should wait: 10
Should wait: 10
Selecting Counties to be Included in the Extract
Should wait: 3
Should wait: 10
Selecting type IDS PROCESS
Should wait: 10
Submit Request

Process done.
Finished in: 00:00:53.99
```

screenshot0
screenshot
counties
dates
submit
done

### CODE REVIEW

File: atd-etl/app/process_cris_request.py [hunk 1]
```diff
@@ -76,30 +76,45 @@ def wait(int):
...
+
+browser.find_by_text('TRAVIS').click()
+
+# Reopen dropdown
+browser.find_by_id('requestCounties').click() 
+browser.find_by_text('WILLIAMSON').click()
+
+# Reopen dropdown
+browser.find_by_id('requestCounties').click() 
+browser.find_by_text('HAYS').click()
```
Comment (sergiogcx) [inline atd-etl/app/process_cris_request.py:99]: Glad you made it work in a more simple way :) far easier to read than injecting javascript and executing 🙏

File: atd-etl/app/process_cris_request.py [hunk 2]
```diff
@@ -76,30 +76,45 @@ def wait(int):
...
-browser.find_by_text("Continue").click()
+browser.find_by_text("Next").click()
 wait(10)
 
 print("Submit Request")
-browser.find_by_text("Submit").click()
+submit_button = browser.find_by_text("Submit Extract Request")
+submit_button.click()
+
+wait(10)
```
Comment (sergiogcx) [inline atd-etl/app/process_cris_request.py:117]: Good call on waiting a few extra seconds to finish off the process. 👍

**Review by sergiogcx — APPROVED** [review]
Looks great! Thank you!

**Review by mddilley — COMMENTED** [review]
Thanks for patching this up, @mateoclarke! This review gave me the chance to check out this script for the first time. Cool stuff here, @sergiogcx! 🤖

**Pull request merged**